src/server.ts

```diff
@@ -21,6 +22,8 @@ if (typeof MONGODB_URI !== 'string') {
 const app = express();
 app.use(helmet());
 
+app.use(cors());
```

Comment:
Let's discuss how we want to handle cross-origin requests.
This will allow requests from any source.
I would propose to limit requests from the Overwolf app only (and localhost in development).
This would be possible by adding `overwolf-extension://{EXTENSION_ID}` as origin.